utils/firestore.js

```diff
@@ -1572,22 +1572,22 @@ const getSpecimenCollections = async (token, siteCode) => {
     return [];
 }
 
-const buildQueryWithFilters = (query, trackingId, endDate, startDate, source, siteCode) => {
-    if (trackingId) {
-        query = query.where('959708259', '==', trackingId);
-    }
-
-    if (endDate) {
-        query = query.where('656548982', '<=', endDate);
+const preQueryBuilder = (filters, query, trackingId, endDate, startDate, source, siteCode) => {
+    if (Object.keys(filters).length === 0 && source === `bptlShippingReport`) {
+        const currentDate = new Date(new Date().getTime()).toISOString();
+        const dateTwoWeeksAgo = new Date(new Date().getTime() - (1209600000)).toISOString();
+        return query = query.where('656548982', '>=', dateTwoWeeksAgo).where('656548982', '<=', currentDate)
```

Comment:
Can be simplified by just returning `query.where('656548982', '>=', dateTwoWeeksAgo).where('656548982', '<=', currentDate)`